tests/rule_based_profiler/parameter_builder/test_value_set_multi_batch_parameter_builder.py

```diff
@@ -1,4 +1,4 @@
-from typing import Any, Collection, List, Optional, Set, cast
+from typing import Any, Collection, Dict, List, Optional, Set, cast
```

Comment:
Is our convention to use the `typing` class if it exists, otherwise use the Python type?

Reply:
Based on our offline conversation I see that Python 3.6 and 3.7 does not support `dict` type parameters.